### PR TITLE
Allow className to be passed to Spinner

### PR DESCRIPTION
**Pull request by ghrlee**
## Author Checklist

- [x] Add unit test(s)
- [x] Update version in package.json (see the [versioning guidelines](https://github.com/LaunchPadLab/opex-public/blob/main/gists/npm-package-guidelines.md#pull-requests-and-deployments))
- [ ] Update documentation (if necessary)
- [ ] Add story to storybook (if necessary)
- [x] Assign dev reviewer

### CODE REVIEW

File: test/indicators/spinner.test.js [hunk 1]
```diff
@@ -22,4 +22,11 @@ test('Spinner filters invalid props', () => {
     <Spinner name="Bob" someInvalidProp="I am not a valid DOM prop" />
   )
   expect(wrapper.find('div#spinner').props().someInvalidProp).not.toBeDefined()
+})
+
+test('Spinner can have custom className', () => {
+  const wrapper = mount(
+    <Spinner className="custom" />
+  )
+  expect(wrapper.find('div.custom').exists()).toBe(true)
```
Comment (chawes13) [inline test/indicators/spinner.test.js:31]: Can you target the spinner via `id` here?

**Review by chawes13 — COMMENTED** [review]
Looks really good, just one small request!

**Comment by chawes13** [comment]
@ghrlee Looks good! Can you update the version in package.json as a patch update? Check out https://semver.org/ for more details on semantic versioning if that's new to you!

**Comment by ghrlee** [comment]
> @ghrlee Looks good! Can you update the version in package.json as a patch update? Check out https://semver.org/ for more details on semantic versioning if that's new to you!

no problem, is there anywhere else that should be updated or just the package.json file?

**Review by chawes13 — APPROVED** [review]
🎉  Great!

**Pull request merged**